source/content/drush.md

```diff
@@ -20,48 +20,80 @@ You can run all of the commands below from Terminus instead of using Drush alias
 For details on managing remote and local Drush versions, see [Managing Drush Versions on Pantheon](/drush-versions).
 
 ## Download Drush Aliases Locally
-Downloading the Pantheon aliases to your local Drush aliases file allows you to run Drush calls against your Pantheon site environments.
+Downloading the Pantheon aliases to your local Drush aliases file allows you to run Drush calls against your Pantheon site environments. Use [Terminus](/terminus/) to download your Drush aliases.
 
-There are two ways to obtain the aliases, either with Terminus or through the Dashboard.
-
-### Download with Terminus, the Pantheon CLI
-Authenticate Terminus with [machine tokens](/machine-tokens/) or your Pantheon Dashboard credentials, then update your local aliases file in a single step:
+Authenticate Terminus with [machine tokens](/docs/machine-tokens/) or your Pantheon Dashboard credentials, then update your local aliases file in a single step:
 
+```bash
+terminus aliases
 ```
-$ terminus aliases
+
+This command will write both Drush 8 and Drush 9 aliases into the directory `$HOME/.drush` for your sites that you are a direct member of. To create aliases for all sites that you can use, including sites that you have access to via team membership, run:
+
+```bash
+terminus aliases --all
 ```
 
-### Download Using the Dashboard
+If you add a site to your account, you will have to download a new copy of your Drush aliases. You do not need to update your Drush aliases when you add new mulitdev environments to your sites.
 
-![Link to Pantheon Drush Aliases](../images/dashboard/drush-aliases.png)
+<Alert type="info" title="Note">
 
-Download your Pantheon site aliases to manually update your local aliases file:
+You must use Drush 8.3.0 or 9.6.0 or later to use Drush aliases directly. Earlier versions are not compatible.
 
-1. From your Pantheon User Dashboard, click **Sites** > **Drush Aliases**
-2. Move the generated `pantheon.aliases.drushrc.php` into your local Drush site-aliases directory (e.g. `$HOME/.drush/site-aliases`).
-3. Clear Drush cache:
+</Alert>
 
- ```
- drush cc drush
- ```
+### Structure of Site Aliases
```

Comment:
Is there related documentation in Drush itself? Maybe we can link to that so we don't have to maintain our own?

Reply:
The Drush 9 alias file format is documented here:

https://github.com/drush-ops/drush/blob/9.x/examples/example.site.yml

The Drush 8 alias file format is documented here:

https://github.com/drush-ops/drush/blob/8.x/examples/example.aliases.drushrc.php

Reply:
I sort of thought it was useful to show them side-by-side for better understanding / comprehension, but we could potentially just cross-reference to the docs to avoid duplicating the info from the Drush docs.